packages/moonstone/Scrollable/ScrollButtons.js

```diff
@@ -279,23 +307,33 @@ class ScrollButtons extends Component {
 
 	onKeyDownPrev = (ev) => {
 		const
+			{focusableScrollbar} = this.props,
 			{nextButtonDisabled} = this.state,
 			{keyCode} = ev;
 
-		if (isPageDown(keyCode) && !nextButtonDisabled) {
-			Spotlight.focus(this.nextButtonNodeRef);
+		if (isPageDown(keyCode)) {
+			if (focusableScrollbar && !nextButtonDisabled) {
+				Spotlight.focus(this.nextButtonNodeRef);
+			} else if (!focusableScrollbar) {
+				this.onClickNext(ev);
```

Comment:
Do we have a requirement to scroll down in this case?

Reply:
Same above.